### PR TITLE
release 1.2.0 final

### PR DESCRIPTION
**Pull request by matthewhanson**
final release

- updated VERSION file
- updated CI
- updated README with ARNs
- updated CHANGELOG

### CODE REVIEW

**Comment by matthewhanson** [comment]
@vincentsarago can you take a look at this and see if there's anything else that should be included in this version?

The current `master` branch is 1.2.0rc2, and those base and python Lambda layers are published, along with the images on Docker Hub. 

The Layer ARNs provided on the `develop` branch don't yet exist, but once they are merged to `master` they would get published.

File: README.md [hunk 1]
```diff
@@ -2,21 +2,19 @@
...
-| 1.0.0     | 2.3.1 |
-| 1.1.0     | 2.4.1 |
-| 1.2.0     | 2.4.2 |
-
-To get the version numbers of all installed packages see the [CHANGELOG](CHANGELOG.md) or the Dockerfile for the specific version.
+| geolambda | GDAL  | Notes |
+| --------- | ----  | |
+| 1.0.0     | 2.3.1 | |
+| 1.1.0     | 2.4.1 | |
+| 1.2.0     | 2.4.2 | Separate Python (3.7.4) image and Lambda Layer added |
```
Comment (vincentsarago) [inline README.md]: This table don't seems to render correctly in Markdown

**Comment by vincentsarago** [comment]
@matthewhanson all seems ok.
Only question, for PREFIX we use `/usr/local/` (https://github.com/developmentseed/geolambda/blob/develop/Dockerfile#L36) so I'm wondering if we are ending up with a layer that has more files that the gdal/proj libs (meaning I think we are packaging files from `lambci/lambda:build-provided` image)

File: README.md (hunk 1 above)
Comment (matthewhanson) [reply README.md]: fixed

**Comment by matthewhanson** [comment]
The /usr/local directory isn't used by system files normally, it's for user built libraries. 
I just checked `lambci/lambda:build-provided` and the /usr/local/lib folder is empty except for `python2.7` and `python3.6` directories, there's no other .so files so the only ones installed are the ones built in the Dockerfile.

**Comment by vincentsarago** [comment]
👌 thus we still endup adding `python2.7` and `python3.6` dirs to the layer right ?

**Comment by matthewhanson** [comment]
No, it's just copying .so files:
```
cp -P ${PREFIX}/lib/*.so* $DEPLOY_DIR/lib/
cp -P ${PREFIX}/lib64/libjpeg*.so* $DEPLOY_DIR/lib/
```
and doesn't recurse into directories

**Pull request merged**